spec/models/user_spec.rb

```diff
@@ -96,6 +96,15 @@
         expect(user.role_flags).to eq(0b0101)
       end
     end
+
+    it do
+      user.update(role_admin: true)
```

Comment:
ok, that means all existing roles in midata will be overwritten. this also means we no longer can give people access to things via orca